### ISSUE TITLE
issue with µ symbol in pdf figure

### ISSUE THREAD

**Issue by dlangenfeld** [UM-R-for-EnvSci-Registered-Student/General-Discussion#11]
I noticed that the "µ" symbol shows up normally in the figures in my .md and .html files but it shows up as a weird different symbol in the .pdf file. Any ideas?

**Comment by madelinestanley** [comment]
I had a similar problem, except it only changed the symbol when I committed it and pushed it back to github. My pdf figure on my computer includes the mu symbol, but not when its hosted in github. I wonder if greek symbols are not recognized the same between github and R.

**Comment by dlangenfeld** [comment]
@madelinestanley I think you are right, the symbol gets lost in translation somewhere between R and GitHub

**Comment by peperg** [comment]
Hello all!

I just uploaded the solutions to the assignment. My µ shows up ok on both the `.png` images embedded in the `.md` as well as in the final `.pdf` image.

This is definitely an issue with the encoding in the transition to `.pdf`.

My suggestion for now would be the use the `cairo_pdf` device as i suggested for when having trouble with fonts. Give it a try and if it doesn't work then we can look at something else. Unfortunately this is a common issue with not a very straight forward solution.

are you both on PC?

Pepe

**Comment by madelinestanley** [comment]
Yes I am on a PC. The document is fine on R and on my computer pdf, however, just not when I push to Github.
